resources/testharness.js

```diff
@@ -1436,11 +1436,28 @@ policies and contribution forms [3].
                 NotAllowedError: 0
             };
 
-            if (!(name in name_code_map)) {
-                throw new AssertionError('Test bug: unrecognized DOMException code "' + code + '" passed to assert_throws()');
-            }
+            var code_name_map = {};
+            Object.entries(name_code_map).forEach(function(keyValue) {
```

Comment:
Even though we're primarily concerned with new language syntax, the use of new built-ins like `Object.entries` could be a problem when used in something as generic (and old) as `assert_throws`. I'm more conservative than most about this, though, and it passed @gsnedders' review, so I'm open to suggestion.

Reply:
Done.